### PR TITLE
[AI] [Inference] Update js-packages.csv

### PR DESCRIPTION
**Pull request by glharper**

### CODE REVIEW

**Review by dargilco — DISMISSED** [review]

File: _data/releases/latest/js-packages.csv [hunk 1]
```diff
@@ -5,7 +5,7 @@
 "@azure/communication-calling-effects","1.0.1","","@azure/communication-calling-effects","Communication","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/communication","NA","NA","client","true","","","","","","","","","","",""
 "@azure/communication-react","1.17.0","","@azure/communication-react","Communication","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/communication","NA","NA","client","true","","","","","","","","","","",""
 "@azure/synapse-access-control","","1.0.0-beta.3","AccessControl","Synapse","synapse","","","client","true","","","","","","","","","synapse-analytics","",""
-"@azure-rest/ai-inference","","1.0.0-beta.1","AI Model Inference","AI","https://github.com/Azure/azure-sdk-for-js/tree/item.Package_item.Version/sdk/ai/ai-inference-rest/","NA","NA","client","true","","","","","","","","","","",""
+"@azure-rest/ai-inference","","1.0.0-beta.1","AI Model Inference","AI","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/ai/ai-inference-rest/","","","client","true","","","","","","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:8]: Why are you changing this to main as opposed to the specific release tag as it currently is?

Comment (weshaggard) [reply _data/releases/latest/js-packages.csv:8]: As for the NA those will go away automatically once the documentation is published and our automation see the docs are present.

Comment (weshaggard) [reply _data/releases/latest/js-packages.csv:8]: @JimSuplizio could you help figure out why the docs are generating? I suspect it has nothing to do with this csv file though.

Comment (JimSuplizio) [reply _data/releases/latest/js-packages.csv:8]: @glharper, this isn't the reason it's not there. I'm going to contact you via email. That being said, this PR does fix the URL to your package however the "NA", "NA" that you're removing isn't something you should be doing. Please add those back but leave the fix for the URL in there.

Comment (glharper) [reply _data/releases/latest/js-packages.csv]: [Updated](https://github.com/Azure/azure-sdk/pull/7771/commits/761036fa5b9bdd15401013c194d0b191d6c87fa4)

**Comment by weshaggard** [comment]
This URL should not be updated it should still contain what it currently does.

**Pull request closed without merging**